### ISSUE TITLE
Refine Open Collective fiscal host offering

### ISSUE THREAD

**Issue by patcon** [hyphacoop/organizing#374]
<sup>_This initial comment is collaborative and open to modification by all._</sup>
## Task Summary

🎟️ **Re-ticketed from:** #336 (also [chat](https://chat.tomesh.net/#/room/#hyphacoop-business-planning:tomesh.net/$nlK87wwVb0Fy2Fa_mFb_BXGWkXCtGTZzY3LxRXFah7s) and [chat](https://chat.tomesh.net/#/room/#hyphacoop-operations:tomesh.net/$qgoVDVAtXrWTlyTjA_Wb7_IZpLj37hIYtrXd8BUdrDI))  
🗣 **Loomio:** N/A  
📅 **Due date:** N/A  
🎯 **Success criteria:** We've moved beyond MVP for this pilot by ...  

This will mainly be a place to track what the reasonable next considerations and tasks are for this internal project.
Notes:
- https://meetings.hypha.coop/2020-11-26-operations.html
- https://meetings.hypha.coop/2020-12-09-fiscal-sponsorship-review-with-sean.html
- https://meetings.hypha.coop/2021-02-12-operations.html
- 2021-02-04 OC hosts https://docs.google.com/document/d/1MxNatiEf1z-WAteLbnB_6kgYGUIbnx235tzw3Uxnqf0/edit
- 2021-03-04 OC hosts https://docs.google.com/document/d/1CNOq63VYHBULnsY01B-_AQwBe1ufuKIQH5nh5pmiVek/edit#

## To Do - Updated 2021-06-17

- [x] dc: update forwarder
- [x] dc: update initiative doc: to get to launch => https://hackmd.io/Kooq3QUcR0CJ2r83OHhemQ#Plan-of-Action
- [x] dc: move over text from emails to template
- [x] ben: followup with TTM project

## To Do

- [x] have call abt pro-bono legal consult on agreement (through connection)
- [x] **Jan** co-work review notes and develop:
   - key actions to wrap up _launch_ phase
   - prelim plan of work / timeline
   - project lead: patcon
- [x] create sample input/output spreadsheet. #todo link
- [x] create operations gdrive
- [x] fix host/collective names to fit on cards
- [x] adjust descriptions to show for key phrases in search
  - canada, ontario, co-operative, coop, Toronto
- [x] remove orphan collective https://opencollective.com/hypha-worker-co-operative
- [x] file ticket about incorrect OC fees on e-transfers https://github.com/opencollective/opencollective/issues/3965
- support docs [PARKED]
    - document timeline expectations for payment
    - document "unpaid" dispute mechanism if doesn't come through. [screenshot](https://user-images.githubusercontent.com/305339/107890617-e39ab600-6ee7-11eb-892b-ea3a00b69e5c.png)
    - cheapest way to get money in. [paypal friends&family](https://www.paypal.com/us/smarthelp/article/what's-the-difference-between-friends-and-family-or-goods-and-services-payments-faq638) reco [from OC slack](https://app.slack.com/client/T0HSL38JD/threads/thread/C0HSLRNVC-1613169466.113500)? e-transfer?
- [x] sync'd member roster on host and root collective (invited missing members)
- [x] create new email forwarder project.opencollective@hypha.coop https://github.com/hyphacoop/organizing-private/pull/116

Related: https://github.com/hyphacoop/organizing-private/issues/115

**Comment by dcwalk** [comment]
From 2020-01-21 ops meeting, interview guide for new collective:

- On our end: 1-2 months to feel ready
- OC interview
- We dont:
    - Offer operational support
    - In-person events: NOT GOOD
    - Hold money more than: $XXX (whole? shouldn't be more than our revenue) For each collective: 5-10k)
    - Help with personal taxes

Questions for us:

- Relation of application to org applying for
- Desire for openness? [We don't have prerequisites]
- Are there types of transactions we don't want (batch? lots of small ones? high volume?)
- Liabilities/risks for us to associating 
- Operational overhead
- uv: who are we expecting to pay out to?

Questions for them:

- Reasons they want to use open collective?
- Who do they serve?
- Who are their associated organizations?
- Size of org, org membership?
- Example of what an expense looks like?
- Volume of transactions (dollar amt)?
- Decision-making structure?
- Risks that they see?
- Current balance? Expected balance?
- Expectations of what the money will be used for?  
    - Re-apply for scope changes?
- Plans for the future? 6 months? 2 years?

- Process in New Collective:
    1. Set up interview and ask them read our page
    2. ??
    3. ??

**Comment by patcon** [comment]
- [2021-02-05 interview with new applicant: Equity4Who](https://meetings.hypha.coop/2021-02-05-equity-4-who.html)
- [2021-02-12 ops meeting](https://meetings.hypha.coop/2021-02-12-operations.html) discussion
  - patcon willing to take lead

**Comment by dcwalk** [comment]
@patcon can we track the todos here?

**Comment by dcwalk** [comment]
@patcon can you update/reframe this issue to just the refining, and "bud out" the new E4W setup elsewhere?

**Comment by patcon** [comment]
Currently asking OCF what would need to be changed in a fork of this website: https://github.com/opencollective/foundation-website

**Comment by dcwalk** [comment]
Not sure I understand the context of needing this app. I think having a
tentative plan for our own roll out is a priority, then seeing how stuff
like this fits in.

**Comment by patcon** [comment]
I agree dc, this is just my place to drop stuff

**Comment by benhylau** [comment]
## Clarifications from Finance on perceived risks

1. On our 10,000 CAD cap and risks associated with high or low cash flow

  - We want to cap the cash flow for a Collective in two ways, (1) while our fiscal hosting is new, and (2) while a Collective is new
  - We want to do this bc if something unexpectedly incurs taxes, we are at an amount we can absorb b/t the Collective and Host
  - If there is some platform error, like the thing with Interac bookkeeping on OC, the Host is absorbing $10 worth of loss not $100+
  - More cash flow generally means more transactions, that requires more bookkeeping from our bookkeeper (takes maybe 1 min to book each outflow), we want to make sure this estimate is correct (not > 1 min), and that we streamline this process further (< 1 min). Notice in here, it's the **number of Tx** that is important, not the amount.
  - Our fees are based on cash flow, which means **if a Collective has high cash flow and low number of Tx and low risk of incurring tax liabilities, that's very good for us** since the 5% will more than cover bookkeeping costs and can help cover other administrative costs of our program which right now is all donated labour

2. On foreign entities and foreign currencies

  - There is little risk for hosting foreign entities from a Finance perspective. In fact, it's more tax risk to host Canadian Collectives
  - In the general case, there is no concept of foreign currency. Everything is in CAD. When someone pays the Collective we receive CAD, and a pay out is made, it's deducted in CAD regardless if what the receiver gets over TransferWise
  - If someone wants to wire in a large sum (e.g. a grant) of a different currency, yes we need to discuss how to hold that money
    - for example, if someone has a 50k USD grant, they can wire it into our CAD account and take ~1% forex hit immediately, that's on the Collective and no risk for us
    - if we decide to **hold the money as USD** then the Host assumes the role of "currency speculator" bc we'd approve a CAD balance while holding an asset that fluctuates. We did this for COMPOST bc it is our project, but we by default are not offering this to Collectives
  - the COMPOST process also incurs additional bookkeeping bc of platform restrictions on OC, but this will change when OC ships https://github.com/opencollective/opencollective/issues/4018 and perhaps allow foreign currency invoices. For now, we only offer this handling as a special case

In other words, speaking with my Finance hat:

>The perfect applicant is one that can start at < $10,000 for first quarter, has clearly articulated tiers of donations that will definitely not trigger taxable events, does not intend to make payments in a way that may trigger employment-like relationships (very low risk of this), with a plan and predictable timeline to ramp up to cash flows > $100,000 while keeping number of payouts low (small # of big payouts rather than large # of small payouts).

**Comment by dcwalk** [comment]
Drafted initiative description: https://hackmd.io/Kooq3QUcR0CJ2r83OHhemQ

**Comment by dcwalk** [comment]
Discussed and developed plan for how to proceed

**Comment by dcwalk** [comment]
Committed the initiative desc. for posterity and closing this out for now. v2 should be developed when folks involved are able to
